docs/reference/esql/esql-syntax.asciidoc

```diff
@@ -36,6 +36,65 @@ source-command | processing-command1 | processing-command2
 ----
 ====
 
+[discrete]
+[[esql-identifiers]]
+==== Identifiers
+
+Unless containing special characters, the identifiers can be used as-are and
+don't require quoting. If they do, however, they must be quoted with
+backticks (```). The special characters are those that have already a semantic
```

Comment:
```suggestion
backticks (+{backtick}+). The special characters are those that have already a semantic
```
Getting a single backtick in an inline code block is not trivial.

Reply:
Nice, thanks!